pkg/hooks/hook_env.go

```diff
@@ -37,7 +55,14 @@ func (h *HookEnv) Pod() (*pods.Pod, error) {
 		return nil, util.Errorf("No pod home given for pod ID %s", id)
 	}
 
-	return pods.NewPod(types.PodID(id), path), nil
+	pod := pods.NewPod(types.PodID(id), path)
+
+	// Spot check that the pod is actually on the disk by looking for the current manifes
```

Comment:
typo: `manifest`
While I'm nitting: maybe we want to say "ensure that the pod is actually…"